corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -55,6 +55,20 @@
     }
   }
 
+  .stripe-repeats {
```

Comment:
I'm not sure, if in your screenshot example, you added a RepeatGroup B directly after RepeatGroupA, would the entire container of RepeatGroupB be shaded grey?

Reply:
It would start the odd/even striping over because their `.gr`s are nested separately, and `nth-of-type` only applies to sibling elements. So this works even if there is a single "group" applying the styling to two or more repeat groups. If there were to be significant change to the html nesting of groups/repeat groups, the style would probably have to be targeted differently.